src/constants/supportedNetworks.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ChainParams,
+  NetworkChains,
+  SupportedNetworks,
+} from "../types/supportedNetworks";
+
+export const SUPPORTED_NETWORKS: SupportedNetworks = {
```

Comment:
Get all chain information from @airswap/constants (as much as is possible)

So SUPPORTED_NETWORKS should be:

```suggestion
export const SUPPORTED_NETWORKS: ChainIds[] = [
  ChainIds.MAINNET,
  ChainIds.AVALANCHE,
  ChainIds.BSC,
  ChainIds.POLYGON,
];```

It has `ChainIds`, `chainNames` and `chainCurrencies` you can import.

Reply:
if it's an array of ChainId's, how would you access the correct index without looping over the array? Wouldn't this be easier?

`export const SUPPORTED_NETWORKS: { [x: string]: ChainIds } = {
  mainnet: ChainIds.MAINNET,
  avalanche: ChainIds.AVALANCHE,
  bac: ChainIds.BSC,
  plygon: ChainIds.POLYGON,
};`

Reply:
Just implement the airswap constants, its up to you how you do it.

Reply:
I consolidated all of the objects I created into 1 huge object called `CHAIN_PARAMS`. This references @airswa/constants where possible